src/foam/nanos/http/LiveScriptBundler.java

```diff
@@ -0,0 +1,396 @@
+/**
```

Comment:
This should be in a separate PR.

Reply:
Oops, this is - not sure how it ended up here